server/graphql-schema.js

```diff
@@ -197,46 +222,91 @@ const graphqlSchema = gql`
     }
 
     input TestPlanReportInput {
-        testPlan: ID!
-        testPlanTarget: ID!
+        testPlanVersionId: ID!
+        testPlanTarget: TestPlanTargetInput!
+    }
+
+    input LocationOfDataInput {
+        testPlanId: ID
+        testPlanVersionId: ID
+        testIndex: Int
+        passThroughIndex: Int
+        testPlanReportId: ID
+        testPlanTargetId: ID
+        browserId: ID
+        browserVersion: String
+        atId: ID
+        atVersion: String
+        testPlanRunId: ID
+        testResultIndex: Int
+        passThroughResultIndex: Int
+    }
+    type LocationOfData {
+        testPlanId: ID
+        testPlanVersionId: ID
+        testIndex: Int
+        passThroughIndex: Int
+        testPlanReportId: ID
+        testPlanTargetId: ID
+        browserId: ID
+        browserVersion: String
+        atId: ID
+        atVersion: String
+        testPlanRunId: ID
+        testResultIndex: Int
+        passThroughResultIndex: Int
+    }
+
+    type PopulatedData {
```

Comment:
nit: `PopulatedData`'s name may make its purpose a bit too ambiguous without looking through the attributes and where it's used as a return type. Maybe just add an additional comment here?